compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/LocalUITraitCollectionState.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.uikit
+
+import androidx.compose.runtime.State
+import androidx.compose.runtime.staticCompositionLocalOf
+import platform.UIKit.UITraitCollection
+
+/**
+ * Composition local for [UITraitCollection]
+ */
+val LocalUITraitCollectionState = staticCompositionLocalOf<State<UITraitCollection>> {
```

Comment:
`LocalUIViewController` is already exposed 👉👈

Reply:
It can be annotated with `@InternalComposeUiApi`. But i found it useful because it is the only way of getting an updated value of it without  [~~костылей~~](https://github.com/JetBrains/compose-multiplatform/issues/3213#issuecomment-1572378546) in Compose-only projects

Reply:
It also have an interface idiom property that can be useful. Not sure if I’m can update at runtime

Reply:
I think - it is fine to have it as a public `val LocalUITraitCollectionState`.
Compose for iOS is still in alpha overall - and we can change this decision in the future.

But, maybe annotation `@InternalComposeUiApi` can also be helpfull. Library authors can also at it to their code.